commands/common.go

```diff
@@ -194,15 +189,18 @@ func (c *config) RemoveTenet(name string, group string) error {
 		return errors.Trace(err)
 	}
 	var tenets []TenetConfig
-	for _, t := range c.AllTenets() {
+	err = errors.Errorf("tenet %q not found", name)
+	for _, t := range g.Tenets {
 		if t.Name != name {
 			tenets = append(tenets, t)
```

Comment:
@waigani I'm not sure if it's possible to be any terser here. Since this is a sort of remove-by-omission you have to let the loop complete, otherwise you'll get e.g. `[1, 2, 3, 4].Remove(3) -> [1, 2]` (If you look at the diff line 150 that's actually also the reason the `break` was removed - it was chopping the tail)

I could use a bool seenTenet instead which might communicate intent more clearly but would actually be a few lines longer